packages/ui-components/src/components/Map/Markers/markerIcons.js

```diff
@@ -285,15 +301,29 @@ export const LEGEND_SHADING_ICON = 'square';
 export const LEGEND_RADIUS_ICON = 'radius';
 export const HIDDEN_ICON = 'hidden';
 
+const scAnchor = (anchor = [0.5 * ICON_BASE_SIZE, 0.5 * ICON_BASE_SIZE], scale = 1) => {
```

Comment:
I can't find where this function is being used? Is it still needed?

Reply:
Ooops 👻